pkg/cli/admin/groups/sync/prune.go

```diff
@@ -5,7 +5,9 @@ import (
 	"fmt"
 
 	"github.com/go-ldap/ldap/v3"
+	ocmdhelpers "github.com/openshift/oc/pkg/helpers/cmd"
```

Comment:
nit: should be in oc/pkg group with the end of imports

Reply:
Do you have an import formatter enabled on the repo? I think you (and anyone making PRs) would appreciate your preferences being automated so these kinds of comments don't need to happen. OpenShift has an import formatter that I've seen other repos use.

Reply:
https://github.com/openshift-eng/openshift-goimports